extensions/README.md

```diff
@@ -103,40 +147,66 @@ If you need to modify the build process, bear in mind that config files are also
-- **--slug**: Specific slug to identify the block that overrides the one generated base don the title.
-- **--description**: Allows to provide a text description of the block.
-- **--keywords**: Provide up to three keywords separated by comma so users  when they search for a block in the editor.
+### Don't worry about dependencies
 
-### Files
+The build takes care of core dependencies for both editor and view -scripts. React, lodash and `@wordpress/*` [dependencies](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md) are externalized and automatically enqueued in PHP for your extension.
 
-All files will be created in a directory under `extensions/blocks/` named after the block title or a specific given slug. For a hypothetical **Jukebox** block, it will create the following files
+Extensions _always_ get [Gutenberg's polyfill scripts](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md#polyfill-scripts) enqueued so you can safely use methods not supported by older browsers such as IE11.
 
-- `extensions/blocks/jukebox/`
-- `extensions/blocks/jukebox/jukebox.php`
-- `extensions/blocks/jukebox/index.js`
-- `extensions/blocks/jukebox/editor.js`
-- `extensions/blocks/jukebox/editor.scss`
-- `extensions/blocks/jukebox/edit.js`
+### Jetpack plugin sidebar
 
-Additionally, the slug of the new block will be added to the `beta` array in the file `extensions/index.json`.
-Since it's added to the beta array, you need to load the beta blocks as explained above to be able to test this block.
+Jetpack adds its own [plugin sidebar](https://wordpress.org/gutenberg/handbook/designers-developers/developers/tutorials/plugin-sidebar-0/plugin-sidebar-1-up-and-running/) to the block editor. You can find it by choosing "Jetpack" from block editor's ellipsis menu or by pressing Jetpack icon in the "pinned plugins" toolbar.
 
-### Examples
+The sidebar itself is always registered in the editor and filled using [Slot Fill](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/slot-fill#readme) mechanism.
 
-`wp jetpack scaffold block "Cool Block"`
+Just render with `JetpackPluginSidebar` component from anywhere in your plugin's code:
 
-`wp jetpack scaffold block "Amazing Rock" --slug="good-music" --description="Rock the best music on your site"`
+```jsx
+import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
 
-`wp jetpack scaffold block "Jukebox" --keywords="music, audio, media"`
+<JetpackPluginSidebar>
+	<PanelBody title={ __( 'My sidebar section', 'jetpack' ) }>
+		<p>Jetpack is Bestpack!</p>
+	</PanelBody>
+</JetpackPluginSidebar>
+```
+
+The sidebar won't show up at all if nothing isn't rendering in the sidebar's "slot".
+
+Remember to be mindful of posts, pages, custom post types, and re-usable block post type.
```

Comment:
Could be a bit clearer. Maybe

```suggestion
Remember to be mindful of what post types you want to enable your sidebar section for: e.g. posts, pages, custom post types, and re-usable block post type.
```

(Also, what's the last one?)

Reply:
Reusable blocks are private custom post types on their own. You can find them via "manage" link in the block picker:

<img width="415" alt="image" src="https://user-images.githubusercontent.com/87168/56416318-231afb80-6299-11e9-87c0-12ae1c233597.png">

`/wp-admin/edit.php?post_type=wp_block`

I'll elaborate more in the doc.